councilmatic_core/management/commands/import_data.py

```diff
@@ -449,32 +456,33 @@ def grab_people(self):
 
     def grab_person_memberships(self, person_id):
         # this grabs a person and all their memberships
-
         url = base_url + '/' + person_id + '/'
-        r = session.get(url)
-        page_json = json.loads(r.text)
-
-        # save image to disk
-        if page_json['image']:
-            r = session.get(page_json['image'], verify=False)
-            if r.status_code == 200:
-                with open((settings.HEADSHOT_PATH + page_json['id'] + ".jpg"), 'wb') as f:
-                    for chunk in r.iter_content(1000):
-                        f.write(chunk)
-                        f.flush()
-
-        page_json['email'] = None
-        for contact_detail in page_json['contact_details']:
-            if contact_detail['type'] == 'email':
-                if contact_detail['value'] != 'mailto:':
-                    page_json['email'] = contact_detail['value']
-
-        page_json['website_url'] = None
-        for link in page_json['links']:
-            if link['note'] == "web site":
-                page_json['website_url'] = link['url']
-
-        return page_json
+        r = self._get_response(url)
+
+        if r:
+            page_json = json.loads(r.text)
+
+            # save image to disk
+            if page_json['image']:
+                r = session.get(page_json['image'], verify=False)
```

Comment:
How come this isn't using the `_get_response` thing?

Reply:
Ah, I overlooked that one....I'll need to account for the `verify=False` argument.